src/core/constraints.jl

```diff
@@ -335,6 +335,111 @@ struct PhaseAngleControlLimit <: ConstraintType end
 struct InterfaceFlowLimit <: ConstraintType end
 struct HVDCFlowCalculationConstraint <: ConstraintType end
 
+"""
+Struct to create the constraint that calculates the Rectifier DC line voltage.
+
+```math
+v_d^r = \\frac{3}{\\pi}N^r \\left( \\sqrt{2}\frac{a^r v_\\text{ac}^r}{t^r}\\cos{\\alpha^r}-X^r I_d \\right)
+```
+"""
+struct HVDCRectifierDCLineVoltageConstraint <: ConstraintType end
```

Comment:
@rodrigomha how does this definitions conflict with the VSC model?

Reply:
The model is completely different for the VSC here

Reply:
Yes, I understand that in some cases the model is different but we should still look into consolidating the variables as much as possible

Reply:
I can take a look if we need to have a constraint here in the VSC, I would suggest replacing this for something like`HVDCFromVoltageConstraint`, that could probably be re-used. I think the approach for now is to use the variables we have and once I start the VSC rewrite, I will look at all the variables and constraints to reuse, and modify accordingly at that point.